core/docker/default/etc/log.properties

```diff
@@ -1,2 +1,4 @@
 # Enable verbose logging from Trino
 #io.trino=DEBUG
+# Disables "provider ... registered in SERVER runtime does not implement any provider interfaces applicable in the SERVER runtime"
+org.glassfish.jersey.internal.inject=ERROR
```

Comment:
I love that we’re getting rid of those logs — they’ve been bugging me for a while.

However, we it shouldn’t require the user to configure this level in the server logging config. It should behave like that out of the box.

Reply:
It cannot be disabled directly in the `Providers` class.

Reply:
Maybe we should disable it programmatically in the server initialization code.

Reply:
Yeah. That makes more sense imo

Reply:
@martint https://github.com/airlift/airlift/pull/1123